composer/core/state.py

```diff
@@ -67,13 +70,26 @@
 
 
 class State(Serializable):
-    """The class used to store the state of the trainer.
+    """The state of the trainer.
```

Comment:
The top of the state docs look like the below - is that right? Seems like those tables shouldn't be there
<img width="954" alt="Screen Shot 2022-02-07 at 5 44 40 PM" src="https://user-images.githubusercontent.com/9918578/152902264-555c3b8a-d755-43cd-8f0c-4339e572c1a6.png">
:

Reply:
These tables will disappear once `__all__` is properly defined on state.